akka-docs/src/test/java/jdocs/stream/javadsl/cookbook/RecipeLoggingElements.java

```diff
@@ -68,14 +68,14 @@ public void workWithLog() throws Exception {
       {
         final Source<String, NotUsed> mySource = Source.from(Arrays.asList("1", "2", "3"));
 
-        final int onElement = Logging.WarningLevel();
-        final int onFinish = Logging.ErrorLevel();
-        final int onFailure = Logging.ErrorLevel();
-
         //#log-custom
         // customise log levels
         mySource.log("before-map")
-          .withAttributes(Attributes.createLogLevels(onElement, onFinish, onFailure))
+          .withAttributes(Attributes.createLogLevels(
+            Logging.WarningLevel(), //onElement
+            Logging.InfoLevel(),    //onFinish
+            Logging.DebugLevel()    //onFailure
```

Comment:
Moving these inside the visible code snippet, for users to easily find how to configure levels by these `Logging` factory methods.

Reply:
👍